tesseract_core/runtime/experimental.py

```diff
@@ -230,6 +234,19 @@ def _strip_output_path(path: Path) -> Path:
 OutputFileReference = Annotated[Path, AfterValidator(_strip_output_path)]
 
 
+def require_file(file_path: PathLike) -> Path:
+    """Require a file to be present at the given path."""
```

Comment:
Please extend the docstring so people would know how to use this feature from looking at the auto-generated docs.